recipes/lipo/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "lipo" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 981706fb20c21c617341996e2e57dec4a369d57aab5bd716431195a6e9b15ebe
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - poetry >=1.0.5
+  run:
+    - python
```

Comment:
Can you also change this to `python >=3.6`

Reply:
Overseen this. Done.